pages/common/autossh.md

```diff
@@ -1,29 +1,29 @@
 # autossh
 
 > Runs, monitors and restarts SSH connections.
```

Comment:
```suggestion
> Run, monitori and restart SSH connections.
```
Don't we usually keep the descriptions in this format?

Reply:
Yes. I didn't notice this 🤷🏻